support-workers/cloud-formation/src/templates/state-machine.yaml

```diff
@@ -68,6 +68,93 @@ States:
           Resource: "${SendAcquisitionEventLambda.Arn}"
           End: True
           {{> retry}}
+
+    # We use this step to pass through state that can then be read to three tier steps
+    - StartAt: ThreeTierPassThrough
+      States:
+        ThreeTierPassThrough:
+          Type: Pass
+          End: True
+    Next: ThreeTierUnpackJsonList
+
+  # Unpack the list of states from the parallel tasks into a single item
+  ThreeTierUnpackJsonList:
+    Type: Pass
+    InputPath: $.[0]
+    Next: ThreeTierShouldCreateSupporterPlusSubscription
+
+  ThreeTierShouldCreateSupporterPlusSubscription:
+    Type: Choice
+    Choices:
+      - Next: ThreeTierUnpackSendThankYouEmailState
+        And:
+          - Variable: $.state.acquisitionData.threeTierCreateSupporterPlusSubscription
+            IsPresent: true
+          - Variable: $.state.acquisitionData.threeTierCreateSupporterPlusSubscription
```

Comment:
This is the part that should avoid us having this run as we never actually send this state down the pipes as yet.

Reply:
Even with the `threeTierCreateSupporterPlusSubscription=true` param

<img width="1182" alt="Screenshot 2024-01-24 at 11 47 09" src="https://github.com/guardian/support-frontend/assets/31692/f58ee14c-4d90-4179-a8d5-221aa254df6d">

The Pipeline skips the three tier branch:
<img width="680" alt="Screenshot 2024-01-24 at 11 51 50" src="https://github.com/guardian/support-frontend/assets/31692/f76965c6-774d-4b6a-995d-ea4920080cd7">